spec.bs

```diff
@@ -543,6 +556,260 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
+      1. TODO: Fetch |leadingBidInfo|'s [=leading bid info/seller reporting result=]'s
+         [=reporting result/report url=].
+      1. Set |sellerDone| to true.
+   1. If |componentSellerDone| is false and |leadingBidInfo|'s
+      [=leading bid info/component seller reporting result=] is not null:
+      1. Let |componentSellerMap| be |leadingBidInfo|'s
+         [=leading bid info/component seller reporting result=]'s
+         [=reporting result/reporting beacon map=].
+      1. If |componentSellerMap| is null, set |componentSellerMap| to an empty [=map=] «[]».
+      1. [=Finalize a reporting destination=] with |reportingMap|,
+         {{FenceReportingDestination/component-seller}}, and |componentSellerMap|.
+      1. TODO: Fetch |leadingBidInfo|'s [=leading bid info/component seller reporting result=]'s
+         [=reporting result/report url=].
+      1. Set |componentSellerDone| to true.
+
+</div>
+
+<div algorithm="create nested configs">
+
+To <dfn>create nested configs</dfn> given [=generated bid/ad component descriptors=]
+|adComponentDescriptors|:
+1. Let |nestedConfigs| be an empty [=list=] «».
+1. If |adComponentDescriptors| is null:
+  1. Return |nestedConfigs|.
+1. [=set/For each=] |adComponentDescriptor| of |adComponentDescriptors|:
+  1. Let |fencedFrameConfig| be a [=fenced frame config=] with the following [=struct/items=]:
+    : [=fenced frame config/mapped url=]
+    :: a [=struct=] with the following [=struct/items=]:
+      : [=mapped url/value=]
+      :: |adComponentDescriptor|'s [=ad descriptor/url=]
+
+      : [=mapped url/visibility=]
+      :: "<a for=visibility>`opaque`</a>"
+
+    : [=fenced frame config/container size=]
+    :: null
+
+    : [=fenced frame config/content size=]
+    :: If |adComponentDescriptor|'s [=ad descriptor/size=] is null, then null. Otherwise, a
+       [=struct=] with the following [=struct/items=]:
+       : [=content size/value=]
+       :: |adComponentDescriptor|'s [=ad descriptor/size=] TODO: Resolve screen-relative sizes and
+          macros and cast this properly.
+
+       : [=content size/visibility=]
+       :: "<a for=visibility>`opaque`</a>"
+
+    : [=fenced frame config/interest group descriptor=]
+    :: null
+
+    : [=fenced frame config/on navigate callback=]
+    :: null
+
+    : [=fenced frame config/effective sandbox flags=]
+    :: a [=struct=] with the following [=struct/items=]:
+      : [=effective sandbox flags/value=]
+      :: TODO: fill this in once fenced frame sandbox flags are more fully specified
+
+      : [=effective sandbox flags/visibility=]
+      :: "<a for=visibility>`opaque`</a>"
+
+    : [=fenced frame config/effective enabled permissions=]
+    :: a [=struct=] with the following [=struct/items=]:
+      : [=effective enabled permissions/value=]
+      :: «"<code><a spec="attribution-reporting-api">attribution-reporting</a></code>",
+          "<code>private-aggregation</code>" (TODO:ref), "<code>shared-storage</code>" (TODO:ref),

```

Comment:
Can you fill in this refs? And probably stack them all at the end of this line instead of incrementally part-way through.

Reply:
I can't fill in the refs, because the respective specs haven't defined them yet.

Reply:
Actually, it looks like it exists for Private Aggregation now (https://patcg-individual-drafts.github.io/private-aggregation-api/#private-aggregation). Maybe it was added since I checked.

Reply:
Oh I see. I thought you were trying to do normative spec biblio references like `[[Private-Aggregation]]`, etc. Makes sense.